betterproto/plugin.py

```diff
@@ -251,11 +193,13 @@ def generate_code(request, response):
                         field_type = f.Type.Name(f.type).lower()[5:]
 
                         field_wraps = ""
-                        if f.type_name.startswith(
-                            ".google.protobuf"
-                        ) and f.type_name.endswith("Value"):
-                            w = f.type_name.split(".").pop()[:-5].upper()
-                            field_wraps = f"betterproto.TYPE_{w}"
+                        match_wrapper = re.match(
+                            "\\.google\\.protobuf\\.(.+)Value", f.type_name
```

Comment:
Using a raw string would be nicer.
```suggestion
                            r"\.google\.protobuf\.(.+)Value", f.type_name
```